cmd/juju/ssh/ssh_jump.go

```diff
@@ -84,6 +95,10 @@ func (p *sshJump) initRun(cmd ModelCommand) error {
 
 // cleanupRun performs cleanup after the SSH proxy run.
 func (p *sshJump) cleanupRun() {
+	if p.knownHostsPath != "" {
+		_ = os.Remove(p.knownHostsPath)
```

Comment:
I'm not sure we need a temp file you know? The file itself isn't temporary just the contents. We could have a static file that truncates known contents, and should we add our own knownhosts lines to it, they are persisted? Kinda like how the sshkeys works?